### ISSUE TITLE
Training Steps and Costs

### ISSUE THREAD

**Issue by daoyuan98** [TheDenk/cogvideox-controlnet#27]
Thank you for the great work!

How many steps did you train the controlnets to get a reasonable result?

**Comment by TheDenk** [comment]
hey, about 40k

**Comment by daoyuan98** [comment]
Thanks for the reply!

I am training a Controlnet with a similar structure on Cogvideo-2b-t2v, but the loss does not decrease after training for 4k steps. It is just fluctuating around 0.18-0.22.  Is it normal?

**Comment by TheDenk** [comment]
Sometimes yes, you have to visualize some examples to be sure
